spec/components/previews/about_page/belief_component_preview.rb

```diff
@@ -0,0 +1,11 @@
+module AboutPage
+  class BeliefComponentPreview < ViewComponent::Preview
+    def with_about_page_collection
+      render(BeliefComponent.with_collection(I18n.t('static_pages.about.beliefs')))
```

Comment:
😍 really happy with how moving content to locale files allows us to reuse it like this.

Reply:
> Looks great to me @ChargrilledChook, just one nit.
> 
> Do you have any ideas on how we can make this visible and known to contributors in the docs and/or elsewhere?

@KevinMulhern Great point. What do you think about giving the readme a bit of a refresh to include some info about tooling / links to some of the resources in the wiki? So not just this, but also things like some of the convenience scripts etc.

Also considering writing up some tickets to flesh this out with more of the existing components -  getting some of the more experienced Rails learners onboard might be good as it's a reasonably safe / simple addition to flesh out the prreviews - thoughts?

Reply:
@ChargrilledChook sounds like a solid idea to me mate. It's been a while since the Readme got some TLC.